packages/ember-metal/lib/index.js

```diff
@@ -306,9 +306,19 @@ Ember.run = run;
 @for Ember
 @private
 */
-Ember.Backburner = Backburner;
-// this is the new go forward, once Ember Data updates to using `_Backburner` we
-// can remove the non-underscored version.
+Ember.Backburner = function() {
+  deprecate(
+    'Usage of Ember.Backburner is deprecated.',
+    false,
+    {
+      id: 'ember-metal.ember-backburner',
+      until: '2.8.0',
+      url: 'http://emberjs.com/deprecations/v2.x/#toc_ember-backburner'
+    }
+  );
+
+  return new Backburner(...arguments);
```

Comment:
Babel is transpiling this to use `Function.prototype.bind` (which is not present in Phantom 1.9).

[Demo here](https://babeljs.io/repl/#?evaluate=true&presets=es2015%2Ces2015-loose%2Cstage-2%2Cstage-3&experimental=true&loose=true&spec=false&playground=false&code=Ember.Backburner%20%3D%20function()%20%7B%0A%20%20deprecate(%0A%20%20%20%20'Usage%20of%20Ember.Backburner%20is%20deprecated.'%2C%0A%20%20%20%20false%2C%0A%20%20%20%20%7B%0A%20%20%20%20%20%20id%3A%20'ember-metal.ember-backburner'%2C%0A%20%20%20%20%20%20until%3A%20'2.8.0'%2C%0A%20%20%20%20%20%20url%3A%20'http%3A%2F%2Femberjs.com%2Fdeprecations%2Fv2.x%2F%23toc_ember-backburner'%0A%20%20%20%20%7D%0A%20%20)%3B%0A%0A%20%20return%20new%20Backburner(...arguments)%3B%0A%7D%3B)

I think you will have to do this manually (aka no spread):

```js
return new Backburner.apply(null, arguments);
```